rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+# Planning for the DCP
+
+## Summary
+
+This RFC adopts and formalizes existing best project management practices and applications from the DCP community to define one process for planning and delivering product increments of the DCP *data as a service* on a regular cadence.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+## Author(s)
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+ 
+ [Brian Raymor](mailto:brianraymor@chanzuckerberg.com)
+
+## Shepherd
+
+[TBD](mailto:username@example.com)
+
+## Motivation
+
+***[Note - remove if approved]***
+*Early versions of this RFC were socialized with the DCP community during a PM call and at the DCP meeting in February 2019* 
+
+One of the **challenges** facing the DCP as it matures was identified in this [Update from HCA Science Governance](https://docs.google.com/presentation/d/13hZ7vmr371mN1YFjFyDN-6c0Ul95NwiJcVsjA3nJFVE/edit#slide=id.p3):
+
+> "A lack of reproducible planning process that prioritizes deliverables and ensures confidence that they will meet the needs of HCA"
+
+### DCP _data as a service_
+
+As a starting point, DCP should think *outside-the-box* and adopt one of the [principles of LeSS](https://less.works/less/principles/whole-product-focus.html) 
+>"Customers don’t buy a part of the product, but the whole product … there is no value in separate parts or half-working parts."
+
+Historically, ownership of the DCP software has been distributed across our canonical architectural diagram with its colorful *think-inside-the-boxes* (components). As a result, DCP planning and development have also been distributed to a high degree, resulting in questions or confusion about DCP priorities relative to component priorities with a subsequent lack of alignment for delivery of product increments, unless externally imposed by requests from HCA Science Governance. 
+
+For example, how does a developer know whether to prioritize research for an operational scoping task assigned by **Technical Architecture**, collaborate on a cross-component RFC surfaced at a DCP quarterly meeting, or develop a component-specific feature? 
+
+To best serve our users and address the concerns of HCA Science Governance, our plans must focus on the needs and priorities of DCP *data as a service* rather than the individual *Chartreuse* or *Polka Dot* components. This requires a DCP-wide project management process with clear priorities and product increments delivered on a regular cadence. 
+
+## Scientific "guardrails" 
+
+As key stackholders, the *Oversight Committee* MUST attend Sprint Reviews on a quarterly basis to offer feedback. 
```

Comment:
OC is not really attending "Sprint Reviews" since those occur at 4 week intervals. Maybe call these "Quarterly Product Reviews"?

Reply:
There was a side discussion with John Marioni and the Project Leads at the DCP F2F. There was general agreement that *Oversight* would attend every third sprint review which would be **special**. I agree that this needs to be clarified in the absence of discussions about _Releases_ in this RFC. Still debating whether to add that section in the absence of a Roadmap RFC to anchor it to ...

Reply:
For the moment, I will not address or resolve this comment further. I'd like to encourage broader discussion about whether we propose a separate quarterly review with _Oversight_ to feature **highlights** from the last three Sprints.

Reply:
I would suggest that highlights from the last three sprints seems more useful for the Oversight group than just what happened in the last sprint otherwise we risk saving demoing of the things which TechArch view are most valuable to Oversight to the meetings they attend.

Also, when stating they must attend, do we have a sense of quorate for their attendance, I doubt we will always get all of them